src/ClientData/include/ClientData/TimerData.h

```diff
@@ -13,7 +13,7 @@
 
 namespace orbit_client_data {
 
-class TrackPaneData final {
+class TimerData final {
```

Comment:
this class has a number of timers not just one. I think the name is misleading... maybe TimerDataManager?

Reply:
Is the data related to the Timers or TimerData. Could be TimersData, but I'm happy with TimerData. We already have a TimerDataManager (creates TimerData) and in the future we will have a TimerDataProvider instead, as well as I am creating a ScopeTreeTimerDataProvider (PR will come soon).

Reply:
I would prefer something more descriptive maybe TrackDataCollection?

Reply:
I already named ScopeTreeTimerData, I should rename that in the same way. What do you think @pierricgimmig ?

Reply:
We already have CallstackData and TracePointData which both represent data on a number of elements. I think TimerData fits well with the current nomenclature.